Cargo.lock

```diff
@@ -4148,6 +4202,49 @@ dependencies = [
  "ws 0.7.9 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "substrate-test-client"
+version = "1.0.0"
+source = "git+https://github.com/yeeco/substrate.git?branch=yee-v1.0#99e45db062bb28b2ce93e41286d89196c5d74ad9"
```

Comment:
all 99e4 are wrong, master is 643d, https://github.com/yeeco/substrate/commits/yee-v1.0